docs/DeveloperGuide.md

```diff
@@ -426,6 +426,24 @@ Use case ends.
 
 <br>
 
+**Use case: Find an event**
```

Comment:
I think Jun Kang added this part in PR#132

Reply:
No worries, I'll add on if applicable :D